tools/ci/docker/linux/Dockerfile

```diff
@@ -191,6 +191,12 @@ RUN cd renesas-tools/source/newlib && \
 RUN cd /tools/renesas-tools/build/gcc && \
   make && make install
 
+# Install Rust and targets supported from NuttX
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y && \
+  $HOME/.cargo/bin/rustup target add thumbv6m-none-eabi && \
```

Comment:
I do not think that it is a good idea to use things like `$HOME` during building a docker container

Reply:
Well, I get your point ... we can change this, if possible (it's the way Rust is installing itself). But, I don't think, that it's the issue.

Assuming, I can change the installation path of Rust ... what would be the right path?

Reply:
done.